metadata-models/src/main/pegasus/com/linkedin/glossary/GlossaryTermInfo.pdl

```diff
@@ -0,0 +1,35 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.Uri
+import com.linkedin.common.GlossaryNodeUrn
+
+/**
+ * Properties associated with a GlossaryTerm
+ */
+record GlossaryTermInfo {
+
+  /**
+   * Definition of business term
+   */
+  definition: string
+
+  /**
+   * Source of the Business Term (INTERNAL or EXTERNAL) with default value as INTERNAL
+   */
+  termSource: string
```

Comment:
if termSource should only have 1 of two values, should it be an enum instead of a string?

Reply:
yes we can have enum for the termSource. Done